plugins/modules/osx_defaults.py

```diff
@@ -71,48 +71,53 @@
 '''
 
 EXAMPLES = r'''
-# TODO: Describe what happens in each example
-
+# Set Boolean Valued Key for Application Domain
 - community.general.osx_defaults:
```

Comment:
Please use `name:` instead of comments, and please do not use title case.
```suggestion
- name: Set boolean valued key for application domain
  community.general.osx_defaults:
```

Reply:
Done